gaia-sdk-javascript/src/api/IdentityOp.ts

```diff
@@ -19,9 +19,9 @@ export class IdentityOp {
         return new DataRef(`gaia://${tenantId}/identities/`, this.client)
             .add(fileName, content, override)
             .pipe(mergeMap(dataRef =>
-                from(this.importIdentity(dataRef.getUri(), tenantId, identityName, override, identityId)
+                defer(() => this.importIdentity(dataRef.getUri(), tenantId, identityName, override, identityId)
```

Comment:
Das ist nicht unbedingt notwendig, weil wir hier schon in einer Lambda Function sind, die erst ausgeführt wird wenn das Observable konsumiert wird.
`dataRef =>
                from(this.importIdentity(dataRef.getUri(), tenantId, identityName, override, identityId)`